app/uk/gov/hmrc/leakdetection/services/AlertingService.scala

```diff
@@ -88,94 +89,32 @@ class AlertingService @Inject()(
+  private def notificationForTeam(messageDetails: MessageDetails, commitInfo: CommitInfo): SlackNotificationRequest =
       SlackNotificationRequest(
-        channelLookup = ChannelLookup.TeamsOfGithubUser(author),
+        channelLookup = ChannelLookup.TeamsOfGithubUser(commitInfo.author),
         messageDetails = messageDetails
       )
 
-    SlackNotificationAndErrorMessage(
-      request    = request,
-      errorMsg   = s"Error sending message to team channels of user: '$author'",
-      commitInfo = commitInfo)
-  }
-
-  private def notificationForAlertChannel(messageDetails: MessageDetails, commitInfo: CommitInfo) = {
-    val request =
+  private def notificationForAlertChannel(messageDetails: MessageDetails, commitInfo: CommitInfo): SlackNotificationRequest =
       SlackNotificationRequest(
         channelLookup  = ChannelLookup.SlackChannel(slackChannels = List(slackConfig.defaultAlertChannel)),
         messageDetails = messageDetails
       )
 
-    SlackNotificationAndErrorMessage(
-      request    = request,
-      errorMsg   = s"Error sending message to default alert channel: '${slackConfig.defaultAlertChannel}'",
-      commitInfo = commitInfo
-    )
-  }
-
-  private def sendSlackMessage(slackNotificationAndErrorMessage: SlackNotificationAndErrorMessage)(
+  private def sendSlackMessage(slackNotificationRequest: SlackNotificationRequest)(
     implicit hc: HeaderCarrier): Future[Unit] =
-    slackConnector.sendMessage(slackNotificationAndErrorMessage.request).map {
+    slackConnector.sendMessage(slackNotificationRequest).map {
       case response if response.hasSentMessages => ()
-      case response =>
-        logger.error(s"Errors sending notification: ${response.errors.mkString("[", ",", "]")}")
-        alertAdminsIfNoSlackChannelFound(response.errors, slackNotificationAndErrorMessage.commitInfo)
-    }
-
-  private def alertAdminsIfNoSlackChannelFound(errors: List[SlackNotificationError], commitInfo: CommitInfo)(
-    implicit hc: HeaderCarrier): Future[Unit] = {
-    val errorsToAlert = errors.filterNot { error =>
-      error.code == "repository_not_found" || error.code == "slack_error"
-    }
-    if (errorsToAlert.nonEmpty) {
-      slackConnector
-        .sendMessage(
-          SlackNotificationRequest(
-            channelLookup = ChannelLookup.SlackChannel(List(slackConfig.adminChannel)),
-            messageDetails = MessageDetails(
-              text        = "LDS failed to deliver slack message to intended channels. Errors are shown below:",
-              username    = slackConfig.username,
-              iconEmoji   = slackConfig.iconEmoji,
-              attachments = errorsToAlert.map(e => Attachment(e.message)) :+ commitInfo.toAttachment,
-              showAttachmentAuthor = false
-            )
-          )
-        )
-        .map(_ => ())
-    } else {
-      Future.unit
+      case response => logger.error(s"Errors sending notification: ${response.errors.mkString("[", ",", "]")}")
```

Comment:
is the response error sufficient to know which request failed, or should we include some info from the request in the message?

Reply:
This was the level of logging that was there previously. We are going to test it in labs, and observe the level of detail in the logs, and will add more details if needed :)